pkg/probe/http/http_test.go

```diff
@@ -39,14 +48,31 @@ func TestHTTPProbeChecker(t *testing.T) {
 
 	prober := New()
 	testCases := []struct {
-		handler func(w http.ResponseWriter)
-		health  probe.Result
-		body    string
+		handler   func(w http.ResponseWriter)
+		health    probe.Result
+		accBodies []string
```

Comment:
can you leave a comment here that explains why this change was necessary? (potentially include something I could grep for like `// go1.5: the error message changed`)